mlos_bench/mlos_bench/tunables/tunable.py

```diff
@@ -546,19 +562,73 @@ def range(self) -> Union[Tuple[int, int], Tuple[float, float]]:
         assert self._range is not None
         return self._range
 
+    @property
+    def span(self) -> Union[int, float]:
```

Comment:
I would rather remove the `.span`, `.quantized_values`, and `.values` properties from the `Tunable`. They are not used anywhere outside of the grid search, and the `Tunable` class has enough accessors to implement that functionality in the optimizer. In general, if something is not used in the scheduler/environment, it should not be part of the `Tunable`.

P.S. I understand that by that logic we should have implemented the distributions/quantization/log stuff outside of the `Tunable`, too - e.g., in some sort of constraints on the optimizer side. But in case of the distributions it was a question of the user convenience - i.e., having all info about the Tunable in one place in JSON. Here I would much rather keep the Tunable intact and move the required logic to teh optimzier

Reply:
Not sure I follow the critique here.  These are basically just introspection properties on the Tunable, and feel pretty natural to me to keep here.

We could export them as utility functions that take a Tunable instead, but why?

Reply:
Moved these to #698, but I think we should keep them.